.github/workflows/ci-poetry.yml

```diff
@@ -18,14 +18,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install Poetry
+        run: pipx install poetry
+
       - uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
-
-      - name: Install Poetry
-        uses: abatilo/actions-poetry@v3.0.0
-        with:
-          poetry-version: 1.2.2
+          cache: pip
```

Comment:
Do we not want `poetry` here?

Reply:
That's a good point. As it stands it will cache the packages used by *this* project (i.e. cookiecutter and pytest) but not the ones used in the generated template project. I did try setting this value to `poetry` but it complained there was no `poetry.lock` in the root folder :disappointed:

Reply:
I think using pip for cache is fine here. There are two different environments being installed, the poetry one is the one that the specification for is build dynamically by cookiecutter within this workflow